source/app/views/forms.erb

```diff
@@ -0,0 +1,49 @@
+<script>
+  $(document).ready(function(){
+    $("#c_one").change(function(){
+      $(".player_one_form").hide("slow");
+      $(".player_two_form").show("slow");
+    });
+    $("#c_two").change(function(){
+      $(".player_two_form").hide("slow");
+      $("#throw").show("slow");
+    });
+  });
+</script>
+<h1>Rock, Paper, Scissors SHOOT!</h1>
+  <img class="img_one" src="/imgs/rock.jpg" alt="paper" height="100" width="100">
+  <img class="img_one" src="/imgs/paper.jpg" alt="paper" height="100" width="100">
+  <img class="img_one" src="/imgs/scissor.jpg" alt="paper" height="100" width="100">
+  <%if @counter==0 %>
+    <h3 style="color: red;">Tie, try again.</h3>
+  <%end%>
+
+  <form action="/newgame" method="post">
+    <p class="player_one_form">Player one select choice:</p>
+    <select id="c_one" class="player_one_form" name="choice1">
+        <option value="default">Choose</option>
+      <%Token.all.each do |v|%>
```

Comment:
So, I actually kindof think this is cool. But this commits the view to listing every single token all the time, and we usually want to decouple the view a bit more than that. So I would recommend passing in a `tokens` variable, either as `@tokens`, or by using `locals: { tokens: Tokens.all }` in your controller.